rules/windows/registry_event/registy_event_crashdump_disabled.yml

```diff
@@ -0,0 +1,20 @@
+title: CrashControl CrashDump Disabled
+id: 2ff692c2-4594-41ec-8fcb-46587de769e0
+status: experimental
+description: Detects disabling the CrashDump per registry as used by HermeticWiper
+author: Tobias Michalski
+date: 2022/02/24
+references:
+    - https://www.sentinelone.com/labs/hermetic-wiper-ukraine-under-attack/
+tags:
```

Comment:
hi `tags` must be a [list](https://github.com/SigmaHQ/sigma/wiki/Specification#tags).
If none simply remove it